### PR TITLE
refacto(Core): Avoid intermediate projection to convert a Extent

### PR DESCRIPTION
**Pull request by gchoqueux**

### CODE REVIEW

File: src/Core/Geographic/Extent.js [hunk 1]
```diff
@@ -72,27 +72,44 @@ Extent.prototype.isTiledCrs = function fnIsTiledCrs() {
     return this._crs.indexOf('WMTS:') == 0 || this._crs == 'TMS';
 };
 
+const s = 20037508.34 * 2;
```
Comment (zarov) [inline src/Core/Geographic/Extent.js]: What is this ? Please add a comment so we don't forget

**Review by zarov — COMMENTED** [review]
Thanks for noticing this ! Apart the comment to add, nothing to add

File: src/Core/Geographic/Extent.js (hunk 1 above)
Comment (gchoqueux) [reply src/Core/Geographic/Extent.js]: added

**Review by zarov — APPROVED** [review]
LGTM, thanks for the comment

**Pull request merged**